spec/SpecHelper.js

```diff
@@ -0,0 +1,15 @@
+beforeEach(function () {
```

Comment:
You don't need this file, unless you write something that helps what you're doing specifically with you jasmine spec files.